css/styles.css

```diff
@@ -50,15 +52,82 @@ li {
   background-color: #e5e5e5;
 }
 
+.close-menu {
```

Comment:
- [ ] You can add some margin to the menu links so your design is as close to the provided design as possible